stubs/paramiko/paramiko/kex_ecdh_nist.pyi

```diff
@@ -2,7 +2,11 @@ import sys
 from _typeshed import ReadableBuffer
 from typing import Callable, Optional, Union
 
-from cryptography.hazmat.primitives.asymmetric.ec2 import EllipticCurve, EllipticCurvePrivateKey, EllipticCurvePublicKey
+from cryptography.hazmat.primitives.asymmetric.ec2 import (  # type: ignore
```

Comment:
Why does this need `# type: ignore`?

Reply:
Because pyright doesn't find this import (probably because it's "hidden" in `stubs/cryptography`). This is something we should look into, but I ignored it for this PR.

Reply:
#5598